src/main/webapp/resources/js/pages/projects/samples-metadata-import2/index.js

```diff
@@ -1,20 +1,32 @@
 import React from "react";
-import { Router } from "@reach/router";
+import { BrowserRouter, Route, Switch } from "react-router-dom";
 import { Provider } from "react-redux";
 import { render } from "react-dom";
 import { SampleMetadataImportUploadFile } from "./components/SampleMetadataImportUploadFile";
 import { SampleMetadataImportMapHeaders } from "./components/SampleMetadataImportMapHeaders";
 import { SampleMetadataImportReview } from "./components/SampleMetadataImportReview";
+import { SampleMetadataImportComplete } from "./components/SampleMetadataImportComplete";
 import { setBaseUrl } from "../../../utilities/url-utilities";
 import store from "./store";
 
```

Comment:
Comment this please

Reply:
Please see 49786d020bc9dedbd03d885f88e05b594c0c5bf0.